features/container/minion/src/main/filtered-resources/etc/system.properties

```diff
@@ -148,3 +148,5 @@ karaf.secured.services = (&(osgi.command.scope=*)(osgi.command.function=*))
 
 # OPENNMS: Enable SNMP4J Log4j logging
 snmp4j.LogFactory=org.snmp4j.log.Log4jLogFactory
+
+${optionals} = confd.system.properties
```

Comment:
Given this is in the default `system.properties` file and not just in the container, maybe we should actually rename it.

Karaf has this stanza in their default configuration, maybe we just reuse it? https://github.com/apache/karaf/blob/master/assemblies/features/base/src/main/resources/resources/etc/system.properties#L25

Note that it's also at the top of the time, and not the bottom, not sure if that makes a difference.

Reply:
Will follow the pattern from the default config.